### DISCUSSION TITLE
Cannot enable SMB in IDH

### DISCUSSION THREAD

**Discussion by brazorfajeje** [Security-Onion-Solutions/securityonion#13472] [Unsupported Versions]
Hi,
I would like to know if it is a known problem that enabling _smb_x_enabled_ in the grid configuration in _idh->opencanary->config_ does not expose the service on port 445 or if there is some problem with my installation.
From opencanary documentation I understand that some configurations are required to allow the component to work, and from securityonion documentation smb is not mentioned among the services that can be enabled, however the configuration is available in grid configuration.
If indeed it is not yet supported, are there plans to implement this service as well?

Thanks!

**Comment by defensivedepth** [comment]
It has not been extensively tested on Security Onion, which is why it is not mentioned in our docs as a supported service. I have created an issue to look into it further: https://github.com/Security-Onion-Solutions/securityonion/issues/13508
